### ISSUE TITLE
BuzzApp CeBuilder Step 1 Issue

### ISSUE THREAD

**Issue by thomasyi79** [inspireui/support#4461]
**_Step 1 (require): describe detail issues & screenshots_**
+ Detail Issues: 
On step 1 in the CeBuilder, I select a folder to save this project to but I get a message that reads: "Currently, CeBuilder hasn't supported this project yet."
+ Product & version:
Buzzapp 1.3.2
+ Flutter (or React Native) version: 
Well this thing is made in react native but I'm not sure about its version. It's probably the latest one since I bought this literally just yesterday.
+ Testing Device/Simulator: 
I'm trying to run this on CeBuilder
+ Screenshot issues (drag the file to attach here): 

![buzzappissue](https://user-images.githubusercontent.com/54558693/89233950-cdb34c80-d5b8-11ea-9cf9-a169ebbab8af.JPG)

**_Step 2 (require): submit proof of purchasing the license on http://verify.inspireui.com_**

So I submitted my proof and this is the page I got after submitting. I legit don't want to wait a full 48 hours so please help me with this. I haven't even been able to start my project because I'm getting an issue/error on step 1.
![submitfile](https://user-images.githubusercontent.com/54558693/89234212-7a8dc980-d5b9-11ea-8444-168444ac1a65.JPG)

**_Important Note:_**
- Please help to ready the support policies before creating a new ticket: https://inspireui.com/support-policies/
- Kindly create only *One Ticket* & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but want to create a new ticket, just do simple step by linking to the previous verified ID (you don't need to submit the form again), for example, #Ticket-ID

Thank you so much for your time 😊

**Comment by minhtruong315** [comment]
Hi @thomasyi79 can u send me package.json file?

**Comment by thomasyi79** [comment]
Hi @minhtruong315 I figured out what was wrong and was able to move past that but the CeBuilder isn't exactly what I expected.. Is it supposed to look like this when trying to edit the layout and whatnot?
![buzzzz](https://user-images.githubusercontent.com/54558693/89317931-15cf7f00-d64c-11ea-95b3-cabacc527a78.JPG)
I thought CeBuilder would be a bit more... graphic/visual? Like a drag and drop type builder, somewhat like the fluxbuilder?
Also, I tried to attach the package.json file to this message but Github doesn't support JSON files.
